xgboost_ray/main.py

```diff
@@ -578,8 +578,9 @@ def predict(self, model: xgb.Booster, data: RayDMatrix, **kwargs):
             self.load_data(data)
         local_data = self._data[data]
 
-        predictions = pd.Series(model.predict(local_data, **kwargs))
-        self._distributed_callbacks.after_predict(self, predictions)
+        predictions = model.predict(local_data, **kwargs)
+        self._distributed_callbacks.after_predict(self,
+                                                  pd.DataFrame(predictions))
```

Comment:
And here convert to Series/DataFrame based on model output?

Reply:
I was thinking about it but I feel that it will actually be easier for users to know that the callback always receives a dataframe and not either one or the other. If they would like to have a series they can easily convert it.